### ISSUE TITLE
Change Setting for Training in Portuguese

### ISSUE THREAD

**Issue by CaioRochaGoes** [NVIDIA/NeMo#2443]
Hi,
I'm doing a training with 10_ASR_CTC_Language_Finetuning, I was wondering if you would have a reference of these settings, ![image](https://user-images.githubusercontent.com/49493322/124511430-7ad7ab80-ddac-11eb-9ed0-45b64c1e180a.png)
which values would be better for a training in Portuguese?

**Comment by titu1994** [comment]
Betas should always be kept the same as the original config of the model. The other hyperparameters have general guidance of all fine tuning models - use a slightly lower learning rate, and use weight decay for regularization as needed. There's no language specific "best set" of hyper parameters

**Comment by CaioRochaGoes** [comment]
Thanks @titu1994! I will do what you say
